test/app/components/sidecar_i18n_component.html.erb

```diff
@@ -0,0 +1,5 @@
+<p class="isolated"><%= t("isolated_translation", default: "This won't be used") %></p>
+<p class="missing"><%= t("translations_component.title", default: "Not the original value") %></p>
+<p class="helpers"><%= helpers.t("translations_component.title", default: "Not the original value") %></p>
+<p class="global"><%= ::I18n.t("translations_component.title", default: "Not the original value") %></p>
```

Comment:
This was a little confusing since I didn't know we had an existing translations file that had a component namespace. Thoughts on using some other namespace that's more general?

Reply:
I cleaned it up a bit, hope is more understandable now.